### ISSUE TITLE
Doesn't work in firefox on termux

### ISSUE THREAD

**Issue by sfantu** [philc/vimium#4541]
On firefox on termux the addon is installed but it just doesn't work.

if i try to debug i get this error:

`Error: Promised response from onMessage listener went out of scope`

It works great in chromium on termux.

**Comment by sfantu** [comment]
Yep ... that is exactly what is going on ... It's the desktop version of Firefox .

**Comment by sfantu** [comment]
Hi i just want to let you know that I just tested Vimium in firefox in termux again and NOW IT WORKS :D. Thank you

**Comment by UncleSnail** [comment]
> Hi i just want to let you know that I just tested Vimium in firefox in termux again and NOW IT WORKS :D. Thank you

Awesome! I'm glad it's working for you now. If you would like, you can close the issue.
